wrappers/src/shared_realm_cs.cpp

```diff
@@ -65,7 +61,7 @@ namespace binding {
 
     CSharpBindingContext::CSharpBindingContext(void* managed_realm_handle) : m_managed_realm_handle(managed_realm_handle)
     {
-        observed_rows = std::vector<ObserverState>();
+        m_observed_rows = std::vector<ObserverState>();
```

Comment:
You don't need to initialize this - the compiler will have added the necessary code to invoke the `std::vector` constructor in the `CSharpBindingContext` constructor.